docs/self-managed/setup/deploy/amazon/amazon-eks/eks-helm.md

```diff
@@ -554,31 +875,31 @@ The following values are required for the OAuth authentication:
 Cluster endpoint=http://localhost:26500
 Client ID='client-id' # retrieve the value from the identity page of your created m2m application
 Client Secret='client-secret' # retrieve the value from the identity page of your created m2m application
-OAuth Token URL=http://localhost:18080/auth/realms/camunda-platform/protocol/openid-connect/token
+OAuth Token URL=http://localhost:8070/auth/realms/camunda-platform/protocol/openid-connect/token
 Audience=zeebe-api # the default for Camunda 8 Self-Managed
 ```
 
 If you want to access the other services and their UI, you can port-forward those as well:
 
 ```shell
 Identity:
-> kubectl port-forward svc/camunda-identity 8080:80
+> kubectl port-forward svc/camunda-identity 8069:80 --namespace camunda
 Operate:
-> kubectl port-forward svc/camunda-operate  8081:80
+> kubectl port-forward svc/camunda-operate  8071:80 --namespace camunda
 Tasklist:
-> kubectl port-forward svc/camunda-tasklist 8082:80
+> kubectl port-forward svc/camunda-tasklist 8072:80 --namespace camunda
 Optimize:
-> kubectl port-forward svc/camunda-optimize 8083:80
+> kubectl port-forward svc/camunda-optimize 8073:80 --namespace camunda
 Connectors:
-> kubectl port-forward svc/camunda-connectors 8088:8080
+> kubectl port-forward svc/camunda-connectors 8078:8080 --namespace camunda
 ```
 
 :::note
 Keycloak must be port-forwarded at all times as it is required to authenticate.
 :::
 
 ```shell
-kubectl port-forward services/camunda-keycloak 18080:80
+kubectl port-forward services/camunda-keycloak 8070:80 --namespace camunda
```

Comment:
same as mentioned above, would keep the ports aligned with the Helm chart output